### ISSUE TITLE
Deprecation Warning for core-js when installing mjml

### ISSUE THREAD

**Issue by mrtopf** [mjmlio/mjml#1761]
**Describe the bug**
When trying to install mjml with `npm install -g mjml` I get the following warning:

```
npm WARN deprecated core-js@2.6.10: core-js@<3.0 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
```

**To Reproduce**
1. Run `sudo npm install -g mjml`
2. See above error.

**Expected behavior**
no warning

**MJML environment (please complete the following information):**
 - OS: MacOS
 - MJML Version 4.5.1
 - MJML tool used: cli,npm

**Additional context**
I tried to track it down and here is parts of the deps tree:

```
├─┬ mjml@4.5.1
│ ├─┬ mjml-accordion@4.5.1
│ │ ├─┬ babel-runtime@6.26.0
│ │ │ ├── core-js@2.6.10
│ │ │ └── regenerator-runtime@0.11.1
│ │ ├── lodash@4.17.15
│ │ └── mjml-core@4.5.1 deduped
```

So it's a dep of babel runtime which is not the most recent version. In version 7 of babel-runtime this is not the case anymore:

https://github.com/babel/babel/blob/v7.7.4/packages/babel-runtime/package.json

So the question might be if this can be upgraded. I don't know which issues that might have down the line though (but I assume eventually this needs to happen anyway).

**Comment by iRyusa** [comment]
Yup, we need to migrate to babel 7

**Comment by nodeg** [comment]
Any news on this? The migration to babel 7 happened last month.

**Comment by iRyusa** [comment]
We still need to do a test publish to see if transpiled version won't have any issue with babel7, when done it will be published with 4.7/5 (version to be defined with custom attribute)
